gptqmodel/utils/audio.py

```diff
@@ -0,0 +1,99 @@
+# SPDX-FileCopyrightText: 2024-2025 ModelCloud.ai
+# SPDX-FileCopyrightText: 2024-2025 qubitium@modelcloud.ai
+# SPDX-License-Identifier: Apache-2.0
+# Contact: qubitium@modelcloud.ai, x.com/qubitium
+
+import base64
+from io import BytesIO
+
+import audioread
+import av
+import librosa
+import numpy as np
```

Comment:
need to add try/catch here since `audioread`/`librosa` is optional pkg requirement